### DISCUSSION TITLE
I am unable to start the self-hosted runner.

### DISCUSSION THREAD

**Discussion by pooneyy** [community/community#128818] [Actions] [answered]
### Select Topic Area

Question

### Body

May I ask what I should do?

![PixPin_2024-06-17_14-40-42](https://github.com/community/community/assets/85266337/f63cd2b9-aa76-4806-9afe-55fc8ce8715d)

**Comment by KovasMcCann** [comment]
Log into your regular user and rerun the program

**Comment by mickeygousset** [comment]
can you just do a `./run.sh`?

**Comment by crohr** [comment, marked as answer]
@pooneyy create a proper user (e.g. `runner`), and launch `run.sh` as this user.

**Reply by pooneyy** [reply]
Thanks！
